tools/dashboard-plugin-tests/src/app/backend/recordedBackend.ts

```diff
@@ -0,0 +1,19 @@
+// (C) 2019-2022 GoodData Corporation
+import { IAnalyticalBackend } from "@gooddata/sdk-backend-spi";
+import { recordedBackend } from "@gooddata/sdk-backend-mockingbird";
+import { ReferenceRecordings } from "@gooddata/reference-workspace";
+import { withLocalDashboardPlugins } from "./withLocalDashboardPlugins";
+import { LocalDashboardPluginsConfig } from "../../types";
+
+export function createRecordedBackend(
+    LocalDashboardPluginsConfig: LocalDashboardPluginsConfig,
+): IAnalyticalBackend {
+    return withLocalDashboardPlugins(
+        recordedBackend(ReferenceRecordings.Recordings, {
+            globalSettings: {
+                enableKDWidgetCustomHeight: true,
```

Comment:
Is this override necessary? Shoul we explicitly set all relevant flags here? So that any settings of any user running this are ignored?

Reply:
Well, that's another good point - we should likely provide all these test settings globally. Not just for recordedBackend, but also for capturing and live tests, to make it consistent.

I see we already have decorator for this - `withCustomWorkspaceSettings`, so I'll use it.